### ISSUE TITLE
Separate unresolved references into JavaScript interop and other symbols categories

### ISSUE THREAD

**Issue by gnl** [cursive-ide/cursive#1526]
Subject says it all, basically like #850 but for ClojureScript/JavaScript - a setting to turn highlighting on and off for both categories would do just fine.

**Comment by cursive-ide** [comment]
Yes, I'm planning to do this, or even turn off the interop warnings altogether since it's almost impossible to resolve reliably in CLJS.

**Comment by zarsky-broad** [comment]
Any progress on this?

**Comment by cursive-ide** [comment]
Yes, I'm actually re-working JS interop right now, and I'll fix this as part of that.

**Comment by zarsky-broad** [comment]
Ah, fantastic! Everyone at the office will be excited to hear that.

**Comment by MrEbbinghaus** [comment]
Anything new about this? I have a namespace which lights up like a Christmas tree because of this.

**Comment by Wegi** [comment]
There still seem to be a lot of not resolvable js interop issues. Would appreciate any progress on it.

**Comment by raymcdermott** [comment]
👍🏻 Even getting rid of the warnings would be preferable to the current visual noise levels.
